app/api/v2/views/users.py

```diff
@@ -47,24 +54,29 @@ def post(self):
             response = Response(json.dumps(
                 invalidpasswordErrorMsg), status=400, mimetype='application/json')
             return response
+        role = data['role']
+
+ 
+
         user = UserModel(
-            data['Firstname'],
-            data['Lastname'],
+            data['firstname'],
+            data['lastname'],
             data['username'],
             data['email'],
             data['password'],
             data['role']
         )
-        cursor.execute(INSERT INTO users(firstname character varying(50) NOT NULL, lastname character varying(50), username character varying(50) NOT NULL, email character varying(50));
-        VALUES('"firstname", "Lastname", "username", "email", "password", "role"')
-        connection.commit()
+        cursor = db_connection().cursor()
+        cursor.execute("INSERT INTO users(firstname, lastname, username, email, password, role) VALUES(%s, %s, %s, %s, %s, %s);",
+                       (firstname, lastname, username, email, password, role))
+        db_connection().commit()
 
 
 class Login(Resource):
     def post(self):
-        data=request.get_json()
-        username=data['username'] or data['email']
-        password=data['password']
+        data = request.get_json()
+        username = data['username'] or data['email']
```

Comment:
decide on one parameter between username and email to be used for login

Reply:
Hello, Boniface,

I will settle with username as that was my primary parameter.

Thank you.

Regards
Steven Ennis